Description: Make build use CPPFLAGS.
 Needed to catch all dpkg-buildflags.
Author: Dominik George <nik@naturalnet.de>
--- a/Makefile
+++ b/Makefile
@@ -5,6 +5,7 @@ SRC 	:= main.c scan-linux.c scan-windows
 
 PATH_AUDIO ?= "./wav"
 
+CFLAGS  += $(CPPFLAGS)
 CFLAGS  += -Wall -Werror 
 CFLAGS  += -DVERSION=\"$(VERSION)\"
 CFLAGS  += -DPATH_AUDIO=\"$(PATH_AUDIO)\"
